tasks/task07/order_repository.py

```diff
@@ -0,0 +1,18 @@
+from order import Order
+import uuid
+
+class OrderRepository:
+    def __init__(self):
+        self.orders = []
+
+    def add(self, order: Order):
+        self.orders.append(order)
+
+    def get(self, order_id: uuid) -> Order:
+        return next(x for x in self.orders if x.order_id == order_id)
+
+    def list(self, n_latest:int = None) -> list:
+       return self.orders if n_latest is None else self.orders[-n_latest:]

```

Comment:
List is python's function name. I would avoid using it as a method name. 

def **list**(self, n_latest:int = None) -> **list**:

Reply:
You probably should check n_latest before passing it as an index.

Reply:
done